packages/config/src/layer2s/canvasconnect.ts

```diff
@@ -0,0 +1,123 @@
+import { ProjectId, UnixTime } from '@l2beat/shared'
+
+import { ProjectDiscovery } from '../discovery/ProjectDiscovery'
+import {
+  getCommittee,
+  getProxyGovernance,
+  getSHARPVerifierContracts,
+  getSHARPVerifierGovernors,
+} from '../discovery/starkware'
+import { delayDescriptionFromString } from '../utils/delayDescription'
+import { formatSeconds } from '../utils/formatSeconds'
+import {
+  CONTRACTS,
+  DATA_AVAILABILITY,
+  EXITS,
+  FORCE_TRANSACTIONS,
+  makeBridgeCompatible,
+  NEW_CRYPTOGRAPHY,
+  NUGGETS,
+  OPERATOR,
+  RISK_VIEW,
+  STATE_CORRECTNESS,
+} from './common'
+import { Layer2 } from './types'
+
+const discovery = new ProjectDiscovery('canvasconnect')
+const delaySeconds = discovery.getContractUpgradeabilityParam(
+  'StarkExchange',
+  'upgradeDelay',
+)
+const delay = formatSeconds(delaySeconds)
+const verifierAddress = discovery.getAddressFromValue(
+  'GpsFactRegistryAdapter',
+  'gpsContract',
+)
+
+export const canvasconnect: Layer2 = {
+  type: 'layer2',
+  id: ProjectId('canvasconnect'),
+  display: {
+    name: 'Canvas Connect',
+    slug: 'canvasconnect',
+    warning:
+      'Canvas Connect is initially open only to whitelisted institutional clients.',
+    description:
+      'Canvas Connect is a Layer 2 ZK network built on the StarkEx technology that aims to offer privacy-focused solutions for the finance and digital economy. It aims to enable the transfer, trade, and investment of tokenized real-world assets, stablecoins, CBDCs, and digital assets with assurances of privacy, low cost, and high speed.',
```

Comment:
Make this less marketingy, get rid of what it aims to and state what it is, Layer2 scaling solution using StarkEx technology developed by StarkWare etc...